src/managed/CommonManaged.props

```diff
@@ -26,4 +32,5 @@
     <Reference Include="System.Core" />
   </ItemGroup>
 
+  <Import Condition="Exists('$(RepoRoot)Tools/versioning.targets')" Project="$(RepoRoot)Tools/versioning.targets" />
```

Comment:
I don't think we need to use this.  The .NET Core SDK has this functionality built into it.  You just need to set the correct properties in the project.

The added benefit is that not only are they set in your assemblies, but they are also set correctly in the output .nupkg.

See [https://github.com/dotnet/sdk/pull/81](https://github.com/dotnet/sdk/pull/81)